they intentionally block qt6 here in this release, but we only have qt6 and it works..
--
diff --git a/Kvantum/kvantummanager/CMakeLists.txt b/Kvantum/kvantummanager/CMakeLists.txt
index 89e2558..161c740 100644
--- a/Kvantum/kvantummanager/CMakeLists.txt
+++ b/Kvantum/kvantummanager/CMakeLists.txt
@@ -1,7 +1,7 @@
 set(QT5_MINIMUM_VERSION "5.15.0")
 set(QT6_MINIMUM_VERSION "6.2.0")
 
-if(NOT ENABLE_QT5)
+if(true)
   set(QT_VERSION_MAJOR 6)
   find_package(Qt6 "${QT6_MINIMUM_VERSION}" REQUIRED COMPONENTS Core Gui Widgets Svg)
 else()
diff --git a/Kvantum/kvantumpreview/CMakeLists.txt b/Kvantum/kvantumpreview/CMakeLists.txt
index ab934b5..d13d85b 100644
--- a/Kvantum/kvantumpreview/CMakeLists.txt
+++ b/Kvantum/kvantumpreview/CMakeLists.txt
@@ -1,7 +1,7 @@
 set(QT5_MINIMUM_VERSION "5.15.0")
 set(QT6_MINIMUM_VERSION "6.2.0")
 
-if(NOT ENABLE_QT5)
+if(true)
   set(QT_VERSION_MAJOR 6)
   find_package(Qt6 "${QT6_MINIMUM_VERSION}" REQUIRED COMPONENTS Core Gui Widgets)
 else()
diff --git a/Kvantum/style/CMakeLists.txt b/Kvantum/style/CMakeLists.txt
index e59cb61..dd470ed 100644
--- a/Kvantum/style/CMakeLists.txt
+++ b/Kvantum/style/CMakeLists.txt
@@ -4,7 +4,7 @@ set(KF5_MINIMUM_VERSION "5.82.0")
 set(KF6_MINIMUM_VERSION "6.0.0")
 
 if(NOT ENABLE_QT4)
-  if(NOT ENABLE_QT5)
+  if(true)
     set(QT_VERSION_MAJOR 6)
     find_package(Qt6 "${QT6_MINIMUM_VERSION}" REQUIRED COMPONENTS Core Gui Widgets Svg)
     if(NOT WITHOUT_KF)
